src/extension.ts

```diff
@@ -210,6 +210,34 @@ async function setupConfig(context: vscode.ExtensionContext) {
   return { configVals, rootPath };
 }
 
+
+function setupPickLabCommand(configVals: ConfigValues, context: vscode.ExtensionContext){
+  context.subscriptions.push(
+    vscode.commands.registerCommand('configActions.updateLab', async () => {
+      const client = ArtificialApollo.getInstance();
+      const response = await client.queryLabs();
+      if (!response) {vscode.window.showErrorMessage('Run Artificial: Sign In Command first to configure connection'); return;}
```

Comment:
I don't like how this shows every time even if the error isn't auth related. Can you make sure to at least stick the error in the output window or something. Would prefer if there was a response code to check.